app/Jobs/RemoveBeatmapsetBestScores.php

```diff
@@ -64,7 +64,12 @@ public function handle()
 
             $class = static::scoreClass($mode);
```

Comment:
let's fix the non-static method while we're at it 👀

Reply:
how does it even work ಠ_ಠ